PatreonClient/PatreonClient.csproj

```diff
@@ -5,7 +5,9 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="System.Text.Json" Version="4.7.1" />
+      <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.6" />
+      <PackageReference Include="System.Net.Http.Json" Version="3.2.1" />
```

Comment:
I don't think we'll need this package unless you are planning to parse the response in 1 go, which we don't atm. That'll need custom json converters for the includes array of the response.

Reply:
That's my next thing on my mind to do.